lm_eval/tasks/goat/_default_matching_template_yaml

```diff
@@ -0,0 +1,32 @@
+dataset_path: deepvk/goat
+output_type: generate_until
+test_split: test
+fewshot_split: dev
+fewshot_config:
+  sampler: first_n
+doc_to_text: "Вопрос: {{input}}\nОтвет должен состоять из последовательности цифр, написанных слитно. Каждая цифра соответствует определённой букве, как в таблице.\n\nОтвет:"
```

Comment:
But now validation of matching tasks is not used in the leaderboard because before validation I need to get rid of html tables in parsed text of matching tasks and replace them with text lists. So I think it'd be better if I remove this file.